evm_loader/lib/src/tracing/tracers/openeth/state_diff.rs

```diff
@@ -0,0 +1,126 @@
+use evm_loader::evm::tracing::{State, States};
+use evm_loader::types::Address;
+use std::collections::BTreeMap;
+use tracing::{debug, info};
+use web3::types::{AccountDiff, Bytes, ChangedType, Diff, StateDiff, H160, H256, U256};
+
+// TODO: Add operator balance diff
```

Comment:
As far as I can see, there is the `balance_diff` method below. Should this comment be removed?

Reply:
I moved the comment to the relevant position and added a Jira ticket number.